portfolio/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -7,4 +7,4 @@
     <!-- prevent unwanted caching when accessing via the web preview server -->
     <include path="/**" expiration="0s" />
   </static-files>
-</appengine-web-app>
+</appengine-web-app>
```

Comment:
Now this is strange. Generally, we should always have a newline at the end of the file (git and github shows special symbol/text if this rule is violated). We are not enforcing this, as normally an auto-formatter plugin should take care of such things. But here - please fix this, as there is change in the file that really brings no meaningful changes and only removes newline.

Reply:
Yes, it happened because originally I added the objectify web filter into this xml file, but it did not work, so I deleted that code, but didn't notice the new line.